ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Suchparameter.md

```diff
@@ -22,15 +24,21 @@ Für die im Rahmen dieses Leitfadens relevanten Typen gelten folgende allgemeine
 
 ### Date/DateTime
 
-Die Präfixe `lt`,`le`,`gt`,`ge`,`eq` MÜSSEN für jeden Suchparameter vom Typ 'date/dateTime' unterstützt werden.
+Die Präfixe `lt`,`gt`,`eq` MÜSSEN für jeden Suchparameter vom Typ 'date/dateTime' unterstützt werden.
+
+Begründung: Die Funktionalität datums-eingeschränkt suchen zu können ist essentiell.
+Hinweis: Die Abfragemöglichkeit arbeitet ungewollten Massendatenabfragen entgegen, da sich sonst Suchen zwangsläufig über den gesamten Zeitraum erstrecken würden.
+
+ Die Präfixe `le`,`ge` KÖNNEN für jeden Suchparameter vom Typ 'date/dateTime' unterstützt werden.
+
 
 **Beispiele**:
 
-```[base]/Patient?birthDate=ge2000-01-01``` <br>
-Suche nach allen Patienten mit einem Geburtsdatum 2000-01-01T00:00 oder später.
+```[base]/Encounter?date=ge2024-01-01&patient=Patient/Test``` <br>
+Suche nach allen Kontakten mit einem Datum ab 2000-01-01T00:00 oder später im Patientenkontext "Test".
```

Comment:
suggestion: in Beispielen verpflichtende Suchparameter verwenden anstatt von optionalen (ge)

Reply:
qe statt gq übernomen